flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/config/MySqlSourceConfigFactory.java

```diff
@@ -317,6 +317,8 @@ public MySqlSourceConfig createConfig(int subtaskId) {
         if (jdbcProperties == null) {
             jdbcProperties = new Properties();
         }
+        // https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-implementation-notes.html
+        jdbcProperties.setProperty("useCursorFetch", "true");
```

Comment:
Move this line near its relative code.

Reply:
Hi @ruanhang1993  Thank you for your advice, but 'jdbcProperties' is initialized here. Do I need to move it here?

Reply:
I think we could initialize the `jdbcProperties` earlier.

Reply:
hi @ruanhang1993 , I think the important code is at the top, but it is not the most important, so I put it at the end